src/test/java/pages/RecentlyPlayedPage.java

```diff
@@ -0,0 +1,35 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RecentlyPlayedPage extends BasePage{
+
+    public RecentlyPlayedPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    //elements
+    static By recentlyPlayedPage = By.cssSelector("a[href='#!/recently-played']");
+    static By AllBtn = By.cssSelector("button.btn-shuffle-all");
+    static By ClearAllBtn = By.cssSelector("button[title='Clear current queue']");
+    static By NoSongsQueuedText = By.cssSelector("div.text");
+
+    public static void clickRecentlyPlayedBtn(){
+        findElement(recentlyPlayedPage).click();
+    }
+    public static void clickAllBtn(){
+        findElement(AllBtn).click();
+    }
+
+    public static void clickClearBtn(){
+        findElement(ClearAllBtn).click();
+    }
+
+    public static WebElement getTextForClearedRecentlyPlayedPage() {
+        return findElement(NoSongsQueuedText);
+    }
```

Comment:
It's best to avoid using **static** to declare page elements so that each instance of the class has its own elements.